tests/dogtag/pytest-ansible/pki/testlib/common/utils.py

```diff
@@ -0,0 +1,562 @@
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
+
+"""
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Description: KRA key cli library
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#   This is the library for kra-key cli tests
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Author: Akshay Adhikari <aadhikar@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2017 Red Hat, Inc. All rights reserved.
```

Comment:
(nitpick) Update date and author?

Reply:
1. I am not sure of updating this, This is all old code which we are moving from downstream to upstream, Should i take up someone else's credit.

Reply:
In that case, update the year and add your name to the list? :)

Reply:
Done.!